### ISSUE TITLE
Disabled buttons in dark mode

### ISSUE THREAD

**Issue by vopani** [h2oai/wave#791]
#### Is your feature request related to a problem? Please describe
Disabled buttons in dark mode are barely noticeable. In a dialog, they are the same colour as the dialog background.

#### Describe the solution you'd like
Better colours?

Dark on Form
<img width="181" alt="dark_form" src="https://user-images.githubusercontent.com/8954217/117406405-a532f700-af2a-11eb-97b5-9405d2eda91c.png">

Dark in Dialog
<img width="217" alt="dark_dialog" src="https://user-images.githubusercontent.com/8954217/117406403-a401ca00-af2a-11eb-9f64-64ce98081eab.png">

Light on Dialog
<img width="219" alt="light_dialog" src="https://user-images.githubusercontent.com/8954217/117406407-a5cb8d80-af2a-11eb-9f7f-5ee9c10a6fd9.png">

**Comment by lo5** [comment]
@mturoci @shanaka-rajitha What's a good solution here? Reduce the contrast in dark mode? Does Fluent provide control over this?

**Comment by shanaka-rajitha** [comment]
@lo5 Yes we will have to reduce the contrast of the main background or starting defining the background color of the button code colors in `HEX` or `RGBA` where the `A` which defines the  opacity to set to be 1.

**Comment by mturoci** [comment]
Fluent uses semantic colors for various component states:
![image](https://user-images.githubusercontent.com/64769322/118108241-0c93ef80-b3e0-11eb-80eb-e804d0c8961c.png)

That means we could either use some palette color (e.g. `neutralPrimary`) or compute high enough contrast color for disabled components and overwrite the defaults. @lo5 @shanaka-rajitha wdyt?

**Comment by mturoci** [comment]
Another possibility is to use text or primary color for disabled states and adjust the opacity (@geomodular suggested adjusting saturation, but this didn't end up well). We can assume both `text` and `primary` colors will have high enough contrast since these are 2 colors user will directly choose and see with comparison to card background. If the contrast will not be high enough, the whole theme will look weird, not just disabled states.

A bit of experimentation with `primary` color:
![image](https://user-images.githubusercontent.com/64769322/118499494-48ec8600-b727-11eb-873a-a76c52520325.png)
![image](https://user-images.githubusercontent.com/64769322/118500317-fbbce400-b727-11eb-8252-d2ec5c9fca42.png)


A bit of experimentation with `text` color:
![image](https://user-images.githubusercontent.com/64769322/118502254-d29d5300-b729-11eb-832d-59f7706a81ba.png)
![image](https://user-images.githubusercontent.com/64769322/118500447-155e2b80-b728-11eb-9bf8-ccc6cceac458.png)

The screenshots present a general idea of approaching this problem, we can adjust the colors as necessary ofc.
